makefile

```diff
@@ -41,13 +41,22 @@ make style
 
 endef
 
+# Custom configuration flags
+GLVIS_CONFIG_MK ?=
```

Comment:
This line seems unnecessary.

Reply:
I guess the purpose is to indicate that this variable can be configured - that's OK.